### ISSUE TITLE
CostModel lookup failed for expression

### ISSUE THREAD

**Issue by mryndzionek** [tlaplus/tlaplus#415]
When model-checking [this](https://github.com/mryndzionek/tlaplus_specs/blob/8e9108e7e8666a57c29487c67262cb95e77c58b4/CigaretteSmokers.tla) spec I get an error:
```
CostModel lookup failed for expression <line 43, col 49 to line 43, col 66 of module CigaretteSmokers>. Reporting costs into <line 31, col 40 to line 31, col 43 of module CigaretteSmokers> instead (Safety and Liveness checking is unaffected. Please report a bug.)
```
This started occurring after adding `ChooseOne` in [line 31](https://github.com/mryndzionek/tlaplus_specs/blob/8e9108e7e8666a57c29487c67262cb95e77c58b4/CigaretteSmokers.tla#L31). I see two similar issues closed/fixed, but this still occurs in 1.6.0

**Comment by lemmy** [comment]
Please check if this is still an issue with https://nightly.tlapl.us/

**Comment by mryndzionek** [comment]
> Please check if this is still an issue with https://nightly.tlapl.us/

Yes, it's still there.

**Comment by lemmy** [comment]
(Stripped) down test case passes but not the original spec.

**Comment by mryndzionek** [comment]
> (Stripped) down test case passes but not the original spec.

Yes, confirmed. Let me know if I can help somehow. Is there maybe a write-up on how to setup debug environment?

**Comment by lemmy** [comment]
IDE setup: https://github.com/tlaplus/tlaplus/tree/master/general/ide

**Comment by lemmy** [comment]
Not a showstopper for 1.7.0.

**Comment by lemmy** [comment]
```
CostModel lookup failed for expression <line 174, col 93 to line 174, col 105 of module e001ClientSupplier>. Reporting costs into <line 305, col 14 to line 305, col 54 of module PWSSemantics> instead (Safety and Liveness checking is unaffected. Please report a bug.)
```


[e001ClientSupplier.WOk3U.zip](https://github.com/tlaplus/tlaplus/files/6077722/e001ClientSupplier.WOk3U.zip)

**Comment by lemmy** [comment]
https://github.com/tlaplus/tlaplus/pull/588#issuecomment-812904626

**Comment by lemmy** [comment]
One more `CostModelCreator` bug: https://github.com/tlaplus/tlaplus/pull/588#issuecomment-817371371
